linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/ResponseFramingFilter.scala

```diff
@@ -0,0 +1,44 @@
+package io.buoyant.linkerd.protocol.http
+
+import com.twitter.finagle._
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.util.Future
+import io.buoyant.linkerd.protocol.FramingException
+
+/**
+ * A filter that fails badly-framed responses
+ */
+class ResponseFramingFilter extends SimpleFilter[Request, Response] {
+  override def apply(
+    request: Request,
+    service: Service[Request, Response]
+  ): Future[Response] =
+    service(request) flatMap { response =>
```

Comment:
Whoops, that was IDEA's fault; it looks like the default config in my new install automatically reformats code on commit. Gotta turn that off!